migrations/mysql/20240603102429_access-tokens_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE access_tokens (
+    id CHAR(36) DEFAULT (UUID()),
+    token_name VARCHAR(100) NOT NULL,
+    token_hash BINARY(32) NOT NULL,
+    created_at TIMESTAMP NOT NULL,
+    revoked_at TIMESTAMP NULL,
```

Comment:
In other place MySQL needed to define a precision that is compactible with `chrono::DateTime`

`registered_at TIMESTAMP(6) NOT NULL,`